### PR TITLE
Ensure that cyclus and cycamore install.py files are the same

### PR DESCRIPTION
**Pull request by scopatz**
Relatively minor change so that the input files are exactly the same.

### CODE REVIEW

**Comment by FlanFlanagan** [comment]
Worked for me. I'll merge in the morning in case anyone else wants to take a look.

**Comment by scopatz** [comment]
Thanks @FlanFlanagan

**Comment by Baaaaam** [comment]
I'll take a look this morning...

File: install.py [hunk 1]
```diff
@@ -171,6 +173,9 @@ def main():
     hdf5 = "the path to the HDF5 libraries directory"
     parser.add_argument('--hdf5_root', help=hdf5)
 
+    cyclus = "the relative path to Cyclus installation directory"
+    parser.add_argument('--cyclus-root', '--cyclus_root', help=cyclus)
```
Comment (Baaaaam) [inline install.py]: Why do we want to provide `cyclus` relative path to `cyclus` installer ?

Comment (scopatz) [inline install.py]: That is ancillary. We just want the install.py files to be the same everywhere. It is too much work to maintain multiple versions, and so the version we get is the superset of all required features.

**Comment by FlanFlanagan** [comment]
I'm going to let @Baaaaam merge this one. Not sure if we wants to fix the relative path thing or not.

**Comment by gonuke** [comment]
If these files are identical across cyclus & cycamore repos, then it will be great if we can figure out how to maintain a single copy! (for another time)..... merging.

Thanks @scopatz

**Comment by scopatz** [comment]
Thanks @gonuke!

**Pull request merged**